webapp/src/js/actions/messages.js

```diff
@@ -42,13 +48,35 @@ angular.module('inboxServices').factory('MessagesActions',
         });
       }
 
+      function markConversationRead() {
```

Comment:
nit: I would expect a function with this name to accept a conversation parameter or alike.... maybe `markSelectedConversationRead()`?

Reply:
Sold.